roles/monitoring_client/templates/node_exporter_web.yml.j2

```diff
@@ -2,4 +2,4 @@ tls_server_config:
   cert_file: node_exporter.crt
   key_file: node_exporter.key
 basic_auth_users:
-  {{ monitoring_client_exporter_username }}: {{ monitoring_client_exporter_password | password_hash('bcrypt') }}
```

Comment:
Using `bcrypt` with `passlib` was causing an exception when using the `password_hash` filter.